packages/react/src/components/Icon/Icon.base.tsx

```diff
@@ -93,6 +93,7 @@ export class IconBase extends React.Component<IIconProps, IIconState> {
             }
           : {})}
         className={classNames.root}
+        translate="no"
```

Comment:
This should probably only be applied if `isImage` is false (in addition to considerations for `accessibleName`)
```suggestion
        translate={isImage || accessibleName ? undefined : 'no'}
```